libs/model/src/emails/GetDigestEmailData.query.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  EnrichedThread,
+  ExternalServiceUserIds,
+  GetDigestEmailData,
+  Query,
+} from '@hicommonwealth/core';
+import { models } from '@hicommonwealth/model';
+import { QueryTypes } from 'sequelize';
+import { z } from 'zod';
+
+export function GetDigestEmailDataQuery(): Query<typeof GetDigestEmailData> {
+  return {
+    ...GetDigestEmailData,
+    auth: [],
+    secure: true,
+    authStrategy: { name: 'authtoken', userId: ExternalServiceUserIds.Knock },
+    body: async () => {
+      const sevenDaysAgo = new Date();
+      sevenDaysAgo.setDate(new Date().getDate() - 7);
+      const threads = await models.sequelize.query<
+        z.infer<typeof EnrichedThread>
```

Comment:
EnrichedThread doesn't accurately describe the data returned, since we have `top_threads.*`. Encriched data should also have the fields of ThreadAttributes.